js/ssai_pulse.js

```diff
@@ -593,7 +621,7 @@ OO.Ads.manager(function(_, $)
      */
     var _appendAdsProxyQueryParameters = _.bind(function(url, adId)
     {
-      var offset = OFFSET_PARAM + OFFSET_VALUE;
```

Comment:
Do we need OFFSET_VALUE anymore? If it is no longer used, we should remove its declaration.